tests/testthat/test-formatter.r

```diff
@@ -8,6 +8,33 @@ test_that("time_format formats hms objects", {
   expect_equal(time_format(format = "%H")(hms::as.hms(a_time, tz = "UTC")), "11")
 })
 
+
+test_that("number format works correctly", {
```

Comment:
Can you please break up in to smaller tests?  Like one for accuracy, one for decimal vs big mark, one for scale, one for prefix/suffix?

Reply:
tests have been simplified and broken into smaller tests